examples/js/controls/OrbitControls.js

```diff
@@ -514,20 +517,48 @@
 
 			}
 
-			function handleMouseWheel( event ) {
-
+			function followZoom(event){
+				let x = (event.offsetX / scope.domElement.clientWidth) * 2 -1,
+				y = -(event.offsetY / scope.domElement.clientHeight) * 2 + 1,
+				v = new Vector3(x, y, 0);
+	
+				v.unproject(scope.object);
+				v.sub(scope.object.position).setLength(scope.zoomSpeed * 10) // it needs more speed here;
```

Comment:
Was this factor of 10 applied to get one example to work?

Change the scale of the scene by a factor of 1/100 and this fudge factor will no longer work. A different approach is needed.